README.md

```diff
@@ -12,9 +12,9 @@ So far, this feels like how the book is gonna play out:
     - [x] RPC
     - [x] REST
     - [x] Query Languages
-  - [ ] API Contracts
-    - [ ] Service Model
-    - [ ] Data Model
+  - [x] API Contracts
+    - [x] Service Model
+    - [x] Data Model
     - [ ] Design First vs Code First
```

Comment:
OOOOOOOOOer. Not sure what the right state of mind will be for this. I think I need to avoid fighting people in my head, but... its hard because I've had this argument so many time I will end up invoking it with whatever I type. 

If anyone has hints on how I should approach this section, please do share.